extensions/amp-a4a/0.1/test/test-a4a-integration.js

```diff
@@ -156,10 +156,10 @@ describe('integration test: a4a', () => {
   it('should fall back to 3p when extractCreative returns empty sig', () => {
     const extractCreativeAndSignatureStub =
         sandbox.stub(MockA4AImpl.prototype, 'extractCreativeAndSignature');
-    extractCreativeAndSignatureStub.onFirstCall().returns({
+    extractCreativeAndSignatureStub.onFirstCall().returns(Promise.resolve({
```

Comment:
This was a bug, `extractCreativeAndSignatureStub` is expected to return a promise. The stub was not returning a promise.